src/components/cl/hier/cl_hier_context.c

```diff
@@ -1,5 +1,6 @@
 /**
  * Copyright (C) Mellanox Technologies Ltd. 2020-2021.  ALL RIGHTS RESERVED.
+ * Copyright (c) Meta Platforms, Inc. and affiliates. 2022.
```

Comment:
we can remove alignment only change - there is no functional change now in this file